frontend_v2/src/app/components/challenge/challengeevaluation/challengeevaluation.component.scss

```diff
@@ -6,6 +6,12 @@
   padding-right: 10px;
 }
 
+.ev-card-panel {
```

Comment:
@burnerlee why are we adding new css class for `ev-card-panel`. Isn't this already present?